libvips/conversion/smartcrop.c

```diff
@@ -451,9 +484,11 @@ vips_smartcrop_init( VipsSmartcrop *smartcrop )
  * @...: %NULL-terminated list of optional named arguments
  *
  * Optional arguments:
- *
+ * 
  * * @interesting: #VipsInteresting to use to find interesting areas (default: #VIPS_INTERESTING_ATTENTION)
- *
+ * * @attention_x: horizontal position of attention centre when using attention based cropping
+ * * @attention_y: vertical position of attention centre when using attention based cropping
```

Comment:
You need to give the type of the optional argument, as unfortunately introspection doesn't deduce these. Change to:

```
 * * @attention_x: %gint, horizontal position of attention centre when using attention based cropping
```